cli/deno_error.rs

```diff
@@ -252,6 +247,8 @@ impl GetErrorKind for DlopenError {
   }
 }
 
+// NOTE(bartlomieju): seems this is necessary - can't use ErrBox here
+// TODO(bartlomieju): ultimately this should be rewritten to `RuntimeError`?
 impl GetErrorKind for dyn AnyError {
   fn kind(&self) -> ErrorKind {
```

Comment:
I don't think this impl is necessary. I suggest:
```rust
fn kind(&self) -> ErrorKind {
  unreachable!()
}
```

Reply:
Not really, tried that and doesn't work. Need to figure this out